src/connection.rs

```diff
@@ -74,7 +74,63 @@ impl Connection {
     ///
     /// # Example
     /// ```
-    /// todo!()
+    /// use zbus::{fdo::DBusProxy, MessageType, MatchRule, Address};
+    /// use futures_lite::future::{block_on, race, yield_now};
+    /// use futures_lite::pin;
+    /// use futures_lite::StreamExt;
+    /// let receive_good_event = async {
+    ///    let connection = atspi::Connection::open().await.expect("Could not open a11y bus.");
+    ///    let object_match_rule = MatchRule::builder()
+    ///        .msg_type(MessageType::Signal)
+    ///        .interface("org.a11y.atspi.Event.Object").expect("Can not build MatchRule with interface org.a11y.atspi.Event.Object")
+    ///        .build();
+    ///    // crates a DBus proxy object using the same connection as the AT-SPI proxy.
+    ///    let dbus_connection = DBusProxy::new(connection.connection()).await.expect("Could not create DBus proxy!");
+    ///    dbus_connection.add_match_rule(object_match_rule).await.expect("Could not create match rule of org.a11y.atspi.Event.Object interface");
+    ///    let a11y_event_stream = connection.event_stream();
+    ///    pin!(a11y_event_stream);
+		///#   let addr_parts = std::process::Command::new("busctl").arg("--user").arg("call").arg("org.a11y.Bus").arg("/org/a11y/bus").arg("org.a11y.Bus").arg("GetAddress").output().expect("Could not get a11y bus address");
+		///#   assert_eq!(addr_parts.status.code().expect("No status code for `busctl` command"), 0, "Status code for first `busctl` command is not 0.");
+		///#   let addr_parts_string = String::from_utf8(addr_parts.stdout).expect("Output of `busctl` command is not in UTF-8 format");
+		///#   let mut addr_iter = addr_parts_string.split_whitespace();
+		///#   addr_iter.next(); // this is to remove the first "s" in the output.
+		///#   let mut addr_str = addr_iter.next().expect("Unable to get a11y bus address from output of `busctl` command").replace("\"", "");
+		///#   let mut create_event_base_cmd = std::process::Command::new("busctl");
+		///#   let mut create_event_cmd = create_event_base_cmd
+		///#       .arg("--address")
```

Comment:
Could we call `busctl` as part of the Actions script?

Reply:
What is an "Actions script"?

Reply:
I am not too well versed in the GitHub `Actions` lingo, but I meant the script we use to have our CI tests run.